tests/suites/test_suite_platform_printf.function

```diff
@@ -31,6 +31,31 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE */
+void printf_long_max(const char *format, long value)
```

Comment:
From my understand, this test is designed to make sure the test framework could process long int value (actually only `LONG_MAX` could be tested) correctly. So the parameters seem to be redundant.

Reply:
Passing `value` is necessary because the main intent of this test is to check that the test framework passes the value correctly.

Passing `format` is not strictly necessary, but this way we're testing the `mbedtls_printf` function and not a possible compiler optimization. Also it makes the structure of the different tests more uniform. Also it may make it easier to generalize the test in the future.